.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: c
+sudo: required
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+      sudo: required
+    - os: linux
+      dist: xenial
```

Comment:
travis doesn't support xenial vm. you'd have to run an `ubuntu:xenial` docker container, but that would only get you xenial's userspace and there'd be no kernel testing.